.github/ISSUE_TEMPLATE/FEATURE_REQUESTS_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+</br>
+ 
+### Is your feature request related to a problem? Please describe.
+(Please present a concise description of the problem to be addressed by this feature request.) 
+</br></br>
+
+### Describe the solution you'd like
+(A clear and concise description of what you want to happen.)
+* Details of the technical implementation
+* Tradeoffs made in design decisions
+* Caveats and considerations for the future
```

Comment:
Same, move this inside the comments block, as people will have to delete it all the time they create an issue, this should be as a hint/info for 'em on how to dress an issue desc.